go/types/leaf_sequence.go

```diff
@@ -138,10 +138,7 @@ func (seq leafSequence) getItem(idx int) sequenceItem {
 }
 
 func (seq leafSequence) WalkRefs(cb RefCallback) {
-	dec, count := seq.decoderSkipToValues()
-	for i := uint64(0); i < count; i++ {
-		dec.readValue().WalkRefs(cb)
-	}
+	walkRefs(seq.valueBytes(), cb)
```

Comment:
This can go on `valueImpl`

Reply:
https://github.com/attic-labs/noms/pull/3762